packages/design-system-dashboard-cli/src/find-ds-components.js

```diff
@@ -128,6 +128,13 @@ function findComponents(searchStrings) {
     usedReactComponentsRegex
   );
 
+  const usedNamedReactComponentsRegex = 
+    /import { ([^;]+) } from '@department-of-veterans-affairs\/component-library'/gms;
+  const usedNamedReactComponents = findUsedReactComponents(
+    vwModules,
+    usedNamedReactComponentsRegex
+  );
```

Comment:
This looks like it won't handle multi-line imports like prettier likes to create:

```js
import {
  Date,
  AlertBox,
  LoadingIndicator,
} from '@department-of-veterans-affairs/component-library';
```
I added that somewhere in `vets-website` and got:

![No Date component in search results](https://user-images.githubusercontent.com/2008881/171698234-5f3a6479-a09f-43b6-910b-8c78746a6145.png)